components/patient-data/impl/src/main/resources/ApplicationResources.properties

```diff
@@ -192,13 +192,21 @@ PhenoTips.ParentalInformationClass_maternal_age_hint=Enter the mother's age in y
 PhenoTips.ParentalInformationClass_paternal_age=Paternal age at EDD (years)
 PhenoTips.ParentalInformationClass_paternal_age_hint=Enter the father's age in years at the estimated date of delivery
 
-PhenoTips.ParentalInformationClass_pregnancy_history__gravida=Gravida
-PhenoTips.ParentalInformationClass_pregnancy_history__para=Para
-PhenoTips.ParentalInformationClass_pregnancy_history__term=Term
-PhenoTips.ParentalInformationClass_pregnancy_history__preterm=Pre-term
-PhenoTips.ParentalInformationClass_pregnancy_history__sab=SAB
-PhenoTips.ParentalInformationClass_pregnancy_history__tab=TAB
-PhenoTips.ParentalInformationClass_pregnancy_history__births=Live births
+phenotips.UIXField.parentalNames=Parents' names
+phenotips.UIXField.maternalName.label=Mother's name
```

Comment:
I think the form would look better if the label was "Mother" instead of "Mother's name" (and same for father).